### PR TITLE
Tasks #102 #32 #33 individual condo

### PR DESCRIPTION
**Pull request by CarlaShawi**
This pr closes Tasks #32, #33 , #102 
- It is for fetching and displaying the proper information from the DB.
- The edit button is only available to the (building Company) and when the condo is Vacant.
- Contact information section displays owner or rented information if available. If the condo is Vacant it displays the company information. 

Done by: @Tharsipa and @CarlaShawi

### CODE REVIEW

File: src/app/pages/individual-condo/individual-condo.component.ts [hunk 1]
```diff
@@ -1,97 +1,181 @@
...
+        this.userInfo = await this.userService.getEmployeeUser(
+          this.condo?.OccupantID || ''
+        );
+      }
+    }
   }
 
+  goBack(): void {
+    // Navigate to previous route
+    this.router.navigateByUrl('./condos');
```
Comment (Guillaume-Lachapelle) [inline src/app/pages/individual-condo/individual-condo.component.ts:92]: Maybe use Angular's Location service to go back to the previous page. Here's how to use it:
`import { Location } from '@angular/common';`
`constructor(private location: Location) { }`

`goBack(): void {
  this.location.back();
}`

**Review by Guillaume-Lachapelle — CHANGES_REQUESTED** [review]
In condo.component.html, right now the image of the condo is a link to the route `/individual-condo`. This isn't necessarily intuitive for users. An option would be to add a `View details` button on the condo to make it obvious for the user.

Maybe something like this under/over the button **Request for ownership**:
` <div>
                <a
                  [routerLink]="['/individual-condo', building.ID, item.ID]"
                  [queryParams]="item"
                >
                  <button class="buttonItem">View Details</button>
                </a>
              </div>`

The rest looks great!

**Review by Guillaume-Lachapelle — APPROVED** [review]

**Review by amczuboka — APPROVED** [review]
Looks good. Checked for a company and public user on both available properties and my properties.

**Review by npiperni — APPROVED** [review]

**Comment by amczuboka** [comment]
Not a huge priority, but some individual-condo's information are accessed via a button on the condo card and others are accessed by pressing the photo. At first, I didn't think we were able to access the info with the condo card without the button. Either change them all to have button or all of them not to have buttons on have a cursor: pointer for the entire condo card.

**Comment by Guillaume-Lachapelle** [comment]
> Not a huge priority, but some individual-condo's information are accessed via a button on the condo card and others are accessed by pressing the photo. At first, I didn't think we were able to access the info with the condo card without the button. Either change them all to have button or all of them not to have buttons on have a cursor: pointer for the entire condo card.

Fixed!

**Review by IsabelleCzuboka — APPROVED** [review]
Looks great

**Review by Ziadsharkos — APPROVED** [review]

**Review by Holyvert — APPROVED** [review]

**Pull request merged**